src/Tribe/Tickets_View.php

```diff
@@ -1069,8 +1069,25 @@ public function get_tickets_block( $post, $echo = true ) {
 			'submit_button_name'          => $submit_button_name,
 			'cart_url'                    => method_exists( $provider, 'get_cart_url' ) ? $provider->get_cart_url() : '',
 			'checkout_url'                => method_exists( $provider, 'get_checkout_url' ) ? $provider->get_checkout_url() : '',
+			'attendees'                   => null,
+			'attendees_total'             => null,
 		];
 
+		// Check to see if attendees list is being hidden or not.
+		if ( class_exists( 'Tribe__Tickets_Plus__Attendees_List', false ) ) {
+			// Handle Event Tickets Plus compatible logic.
+			$hide_attendee_list_optout = Tribe__Tickets_Plus__Attendees_List::is_hidden_on( $post_id );
+		} else {
+			// Handle Event Tickets logic.
+			$hide_attendee_list_optout = \Tribe\Tickets\Events\Attendees_List::is_hidden_on( $post_id );
+		}
+		// If we are not hiding the attendees output, than grab the data.
+		if ( ! $hide_attendee_list_optout ) {
+			$attendees_list            = tribe( 'tickets.events.attendees-list' );
+			$args[ 'attendees' ]       = $attendees_list->get_attendees_for_post( $post_id );
+			$args[ 'attendees_total' ] = $attendees_list->get_attendance_counts( $post_id );
```

Comment:
The spacing on the array keys should only be applied when the key is a variable.

```suggestion
			$args['attendees']       = $attendees_list->get_attendees_for_post( $post_id );
			$args['attendees_total'] = $attendees_list->get_attendance_counts( $post_id );
```